src/drawer.canvas.js

```diff
@@ -48,6 +48,25 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.Canvas, {
 
         this.clearWave();
     },
+    
+    updateWidthnHeight: function () {
```

Comment:
Hi! That's nice that you work on this!
Can you please show us a HTML exemple of how it look like ? Thanks!